Resources/Prototypes/Entities/Objects/Misc/broken_bottle.yml

```diff
@@ -0,0 +1,21 @@
+- type: entity
+  name: broken bottle
+  parent: BaseItem 
+  id: BrokenBottle
+  description: In Space Glasgow this is called a "conversation starter"
```

Comment:
the quotes might need back slashes?

Reply:
it showed ok in the description but actually i think its better to remove them just in case, the message would be the same